pkg/trtl/server_test.go

```diff
@@ -211,14 +224,36 @@ func (s *trtlTestSuite) setupServers() (err error) {
 	}
 
 	// Create a bufconn listener(s) so that there are no actual network requests
-	s.grpc = bufconn.New(bufSize)
+	s.grpc = bufconn.New(bufSize, "")
 
 	// Run the test server without signals, background routines or maintenance mode checks
 	// TODO: do we need to check if there was an error when starting run?
 	go s.trtl.Run(s.grpc.Listener)
 	return nil
 }
 
+// creates the RemoteTrtl to be used for testing client connections to a remote trtl
+// peer
+func (s *trtlTestSuite) setupRemoteTrtl() (err error) {
```

Comment:
Could we update the comment to say that the connection to Trtl is always over mTLS to create as realistic a connection as possible and that the server certs in the `testdata` were generated by the `certs` command?